src/app/layout.py

```diff
@@ -0,0 +1,150 @@
+import dash
+from dash import html
+from dash import dcc
+import dash_bootstrap_components as dbc
+import dash_alternative_viz as dav
```

Comment:
Also, I noticed that we are importing `dash_alternative_viz` but we don't seem to be using it?